docs/user/rest-api.rst

```diff
@@ -1189,3 +1189,58 @@ Revoke Cert
 .. code-block:: text
 
     POST /api/v1/controller/cert/{id}/revoke/
+
+.. _device_whois_details:
+
+WHOIS Details
+~~~~~~~~~~~~~
+
+.. note::
+
+    This feature is required to be enabled in the settings, see
+    :ref:`OPENWISP_CONTROLLER_WHOIS_ENABLED`.
+
+WHOIS details of a device can be retrieved by either using the device list
+endpoint or device detail endpoint. The list endpoint will contain brief
+details while the detail endpoint will contain all the WHOIS details.
+Example for brief details:
+
+.. code-block:: json
```

Comment:
This is part of the device endpoint right?

We should update that endpoint to mention it will display the whois info if the whois lookup feature is enabled, nothing more, it can be a lot shorter than this and there's no need to include the example here (we delegate that to the live API docs).